libkbfs/block_journal.go

```diff
@@ -254,6 +279,8 @@ func (j *blockJournal) readJournal(ctx context.Context) (
 			}
 		}
 	}
+	j.log.CDebugf(ctx, "Found %d block bytes in the journal", blockBytes)
+	j.blockBytes = blockBytes
```

Comment:
ah, i think this function should return `blockBytes` and let the caller set it on `j` (like it does with `refs`).